### DISCUSSION TITLE
Append when using uppercase register

### DISCUSSION THREAD

**Discussion by BioBox** [helix-editor/helix#9192] [Ideas] [answered]
A nice feature of vim was that you could **append to a register** if you use its uppercase counterpart; this isn't in helix. Even when I try to select a few specific lines and save it to a register, it will only paste the first line.

Would this functionality be welcome? Should I open an issue for this? What are your thoughts?

**Comment by the-mikedavis** [comment, marked as answer]
You can use `*` to append to a register (using `"<register>` to select which register you want to append to). `y` will yank to the register and replace all values instead.

In order to print multiple values out of a register, you will need as many selections, so if you want to print 3 values you will need 3 selections.
